benchmarks/logreg/solvers/lightning.py

```diff
@@ -9,11 +9,11 @@
 class Solver(BaseSolver):
     name = 'Lightning'
 
-    install_cmd = 'pip'
-    requirements = ['sklearn-contrib-lightning']
+    install_cmd = 'conda'
+    requirements = ['pip:sklearn-contrib-lightning']
     requirements_import = ['lightning']
     requirements_install = [
-        'git+https://github.com/scikit-learn-contrib/lightning.git'
+        'pip:git+https://github.com/scikit-learn-contrib/lightning.git'
```

Comment:
to be honest I don't really understand why you need 3 entries for one package. Why a single line is not enough? As if you would pass an env yml or req .txt file.

Reply:
Part of it is legacy and part is because package import name is not always the same as the name on Pypi. We need the import name to do `python -c "import import_name"` to check if the install ran. But maybe a solution would be to rely on `safe_import` to check install with `python -c "from benchmarks.benchname.solvers import solvername; solver_import = getattr(solvername, 'solver_import', None); assert solver_import is None or not solver_import.failed_import"`.

Reply:
yes I think we could rely on the safe_import in solver file. It would simplify things

Reply:
Now we have only two fields, requirements and requirements_install